concent_api/middleman/middleman_server.py

```diff
@@ -148,21 +150,26 @@ def _close_middleman(self) -> None:
             await response_queue.join()  # 3. wait until items from response queue are processed (sent back to Concent)
             response_consumer_task.cancel()
 
+        except asyncio.CancelledError:
```

Comment:
Won't that be catch in the next `except`? Or we just don't want to extra log it?

Reply:
it is added as a measure of avoiding sentry crash logs in case where some `asyncio.Task` is being interrupted by `CTRL-C`